### PR TITLE
Added doc for using workload alertmanager with an app

### PR DESCRIPTION
**Pull request by talha0324**

### CODE REVIEW

**Comment by stakater-user** [comment]
@talha0324 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-131-0a1c1ab1`

File: content/sre/alerting/workload-application-alerts.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# Customer workload Alerting
+
+Stakater Agility Platform provides seperate alerting for customer workloads that offer individual configuration for each new application a customer defines.
```
Comment (rasheedamir) [inline content/sre/alerting/workload-application-alerts.md]: App Agility Platform

File: content/sre/alerting/workload-application-alerts.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+Having alerts for your new application can consist of 3 major blocks:
```
Comment (rasheedamir) [inline content/sre/alerting/workload-application-alerts.md]: major blocks doesn't seem to be the right word

plz add numbering like e.g. step # 1, step # 2, etc.

File: content/sre/alerting/workload-application-alerts.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    stakater.com/workload-monitoring: 'true'
+  name: {NAME}
+~~~
+
+
+## Writing an AlertmanagerConfig for your Applicaiton
```
Comment (rasheedamir) [inline content/sre/alerting/workload-application-alerts.md]: @talha0324 we should mention that our application chart has support as well

File: content/sre/alerting/workload-application-alerts.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
+# Customer workload Alerting
+
+Stakater Agility Platform provides seperate alerting for customer workloads that offer individual configuration for each new application a customer defines.
+
+Having alerts for your new application can consist of 3 major blocks:
+- Application with a `ServiceMonitor/PodMonitor`
```
Comment (rasheedamir) [inline content/sre/alerting/workload-application-alerts.md:6]: we should provide example of each

File: content/sre/alerting/workload-application-alerts.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+    groupWait: 30s
+    groupInterval: 5m
+    repeatInterval: 12h
+  receivers:
+  - name: "null"
+~~~
+
+Now the label `alertmanagerConfig: "workload"` enables it to be picked up by the Alertmanager of workload monitoring offered by Stakater Agility Platform and you can direct this alert anywhere you like pagerduty or slack. Also you cannot change this label. You need to use this specific label to get this configuration picked up by the alertmanager.
+
+## Defining a PrometheusRule for your app
```
Comment (rasheedamir) [inline content/sre/alerting/workload-application-alerts.md]: @talha0324 we need to add support for PrometheusRule in our application helm chart and should provide reference here as well

**Comment by stakater-user** [comment]
@talha0324 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-131-be6d11fa`

**Comment by stakater-user** [comment]
@talha0324 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-131-ef77c635`

**Review by rasheedamir — APPROVED** [review]

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.78`

**Pull request merged**